opm/output/data/Aquifer.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright 2019 Statoil ASA.
```

Comment:
Equinor?

Reply:
How did that happen?  Thanks for noticing.  I'll fix it on the next update.